### ISSUE TITLE
Mail Notifications

### ISSUE THREAD

**Issue by Jonno232** [peterhajas/MobileNotifier#151]
Hi,
Not sure I am doing something wrong I don't get any mail notifications. I have four mail accounts setup and I get no notifications. Running ios 4.3.2 on iPhone 4. 

Cheers.

**Comment by btaroli** [comment]
I don't know whether it's supposed to do that, but I can definitely confirm that it doesn't happen for me. I don't recall that email triggers "alerts" -- basically, it's my understanding that MobileNotifier traps events that would cause the classic "close/reply" pop-ups in iOS. And, honestly, I'd be afraid how often MN would be trapping new email for me. If such a feature is added, I'd want a way to turn it off. :-))

If you're looking for a nice way to be visually notified that new mail exists (in your inbox(es), anyway) I'd suggest the Notifier+ jailbreak app. :) In fact, I'd love it if MN could add an icon to the structure that Notifier+ provides, in case I miss a pop-up and don't want to poll the MN dashboard. :-)

**Comment by snum** [comment]
MobileNotifier + Email Enhancer (with pop-up option selected) will do what Jonno232 is asking for. I'm unsure if Notifier+ will, as I've never used it before.

**Comment by btaroli** [comment]
Notifier+ merely puts an icon in the statusbar for awaiting emails. I actually like the statusbar used in this fashion, because I'm happy to know, at a glace, that there are emails there. I don't necessarily need a shopping list of them, unless I just open up the mail app. I guess the point being that with most other notifications, I can choose whether "alerts" are generated. I guess so long as I have the same option for email then I'd be happy. ;-) I actually used to do the same with SMS... I disabled pop-up alerts and went solely with the badge, to that Notifier+ could just show me there were some waiting. In fact, I still use Notifier+ with MobileNotifier for exactly the reason that I don't have to tap anything to see if stuff is waiting, and I get a vague idea of which stuff it might be (SMS, IM, email, phone calls/VM, etc).

**Comment by snum** [comment]
Agreed. I guess it depends on the # of alerts you receive for certain apps. I use iRealSMS and therefore don't get SMS alerts in MN, which would be a ton (and annoying). I don't receive a high volume of email for it to be annoying in MN though. Maybe an option in MN to configure what apps it alerts on would be useful (assuming someone hasn't already requested it). I know you could disable the alerts in the Settings/Notifications section for each individual app, but maybe there's instances where you'd still want to be alerted (via an obtrusive popup), even though that's what MN is trying to prevent. I'd say badges do this, but you don't always have all apps visibly noticeable where you can see the badges (or badges are disabled in folders, etc).

**Comment by Kaizersoze** [comment]
I think MobileNotifier, for the sake of clarity, should address all possible events of our precious (calendar invitations and alerts, missed calls, voicemails, SMS / MMS, emails, notifications of applications). Thus, we no longer need several tweaks and MobileNotifier become a reference ... I think...unless IOS 5 offers a similar solution ;-)

**Comment by peterhajas** [comment]
Email notifications don't currently work, but they're in progress.

If you'd like separate notifications, please either comment on existing issues for them or (if none exist) create ones.
